### ISSUE TITLE
Body Agenda Item is a List. 

### ISSUE THREAD

**Issue by hungrymonkey** [hackla-engage/engage-backend#131]
May I ask why should the body be a list. I think a Map is more appropriate.

```
curl -X GET  https://backend.engage.town/api/tag/Responsive%20Government/agenda/items/

{
            "agenda": 57,
            "agenda_item_id": "2856",
            "body": [
                "Executive Summary",
                "The Downtown Real-Time Parking Project is identified in the City\u2019s Downtown Community Plan as a means to reduce traffic congestion and help drivers more quickly find available parking.  This project will develop construction-ready plans, specifications, and estimates for real-time parking and traveler information signs for motorists traveling to Santa Monica\u2019s Downtown parking garages adjacent to 2nd Street, 4th Street, and the Pier that conform to the City\u2019s wayfinding standards. Staff recommends Transpo Group, Inc. prepare the engineering design and construction documents in an amount not to exceed amount of $105,000 over a three-year period.",
                "Background",
                "In 2014, the City completed the Real-Time Beach Parking Project that installed 21 digital signs that direct motorist to the beach parking lots.  These signs operate daily to display wayfinding messages and lot occupancy and space availability on a real-time basis.",
                "In 2016, City staff began coordinating with key Downtown stakeholders and other City Departments to explore the expansion of the current real-time system into the Downtown.  Discussion topics included location, size, and aesthetics of the proposed signs.  Several meetings were held including staff from Downtown Santa Monica Inc., Santa Monica Place (Macerich), and Santa Monica Pier Corporation.",
                "In 2017, the City completed the installation of 100 static wayfinding signs in Downtown.  The signs vary in size and conform to the City\u2019s latest wayfinding sign standards.",
                "Discussion",
                "This project involves the engineering design of two types of dynamic message signs (DMS) to be located within the Downtown area bounded by Ocean Avenue, California Avenue, Pico Boulevard, and 6th Street. Both types of signs will conform to the City\u2019s latest standard for wayfinding signs.  The first type is a small, two-sided DMS proposed at the entrance to seven Downtown parking garages indicated in Attachment A. The second type of DMS is a full matrix sign that can display multiple lines of text and graphics.  These DMS will be placed at 18 strategic locations near City entrances indicated in Attachment B.  The DMS will be integrated with the City\u2019s parking availability system and display messages directing motorists from the freeway off-ramps to the nearest available parking facilities based on real-time occupancy conditions.",
                "The DMS will also have the capability to display custom messages and will assist the management and diversion of traffic during special events, planned closures, or unexpected incidents.  The final locations for sign installation will be determined in coordination with other City departments, including Community and Cultural Services, Finance, Information Services, and Public Works.",
                "Consultant Selection",
                "On October 17, 2017, the City published a Request for Proposal (RFP) to obtain engineering design services for the Downtown Real-Time Parking Project.  The RFP was posted on the City\u2019s on-line bidding site, and notices were advertised in the Santa Monica Daily Press in accordance with the City Charter and Municipal Code provisions.  41 vendors downloaded the RFP. Four proposals were received on November 9, 2017 from the firms listed below.",
                "\u00b7         Advantec Consulting Engineers",
                "\u00b7         Kimley-Horn and Associates",
                "\u00b7         Michael Baker International",
                "\u00b7         Transpo Group Inc.",
                "Proposals were evaluated based on the criteria in SMMC 2.24.073 and as outlined in the RFP, which included Experience/Technical Competence, Ability to Meet Work Plan and Timelines, Project Approach, Cost of Services and Cost Control, Stability/ References, and the Value Added/ Breadth of Service.",
                "In January 2017, the selection committee, comprised of staff from Planning and Community Development, recommended Transpo Group, Inc. to provide the engineering design services for this project. Transpo Group demonstrated extensive knowledge and experience with real-time parking projects, and similar capital improvement projects in the City of Seattle, Puget Sound Transit, and San Francisco. Transpo Group also demonstrated through past projects its ability to offer project design on schedule and at a competitive price. Based on these criteria, Transpo Group is recommended as the best qualified firm.",
                "Financial Impacts and Budget Actions",
                "The contract to be awarded to Transpo Group Inc., is for an amount not to exceed $105,000. Funds of $105,000 are available in the FY 2017-18 Capital Improvement Program budget. The contract will be charged to account C010994.589000 \u2013 Parking Guidance System."
            ],
            "department": "Traffic Management (PCD)",
            "id": 926,
            "meeting_time": 1518571800,
            "recommendations": [
                {
                    "agenda_item": 926,
                    "id": 926,
                    "recommendation": [
                        "1.     Award RFP#141 to Transpo Group, Inc., a Washington-based company for Engineering Design services for the Downtown Real-Time Parking Signs Project.",
                        "2.     Authorize the City Manager to negotiate and execute an agreement with Transpo Group, Inc., in an amount not to exceed $94,730 for one year, with two-additional one-year renewal options in the subtotal amount of $10,270, on the same terms and conditions for a total amount not to exceed $105,000 over a three-year period, with future year funding contingent on Council budget approval."
                    ]
                }
            ],
            "sponsors": null,
            "title": "Award Contract to Transpo Group for Engineering Design Services for the Downtown Real-Time Parking Project"
        },
```

**Comment by hungrymonkey** [comment]
```
curl -X GET  https://backend.engage.town/api/agendas/ -H "accept: application/json"
{
                    "agenda": 62,
                    "agenda_item_id": "3211",
                    "body": [
                        "Summary",
                        "Airfield lighting is critical to safe operations at the Santa Monica Municipal Airport (SMO). A recent inspection of the lighting cables showed that certain segments have reached the end of their useful life and need immediate replacement. Staff recommends Bales Engineering Contractors, Inc. (Bales Engineering) to provide construction services for the SMO \u2013 Cable Replacement of Airfield Lighting Project (Project).",
                        "Discussion",
                        "Upon the completion of the Runway Shortening Project in December 2017, an overall inspection of the runway by an airfield lighting contractor revealed that cabling, which was not part of the original project scope, needs to be replaced. Any cable failure would result in the closure of SMO at night and during inclement weather until repairs are completed. To prevent any potential runway lighting issues, an estimated 30,000 linear feet of cable needs to be replaced. The cable is housed in a subsurface conduit, and the condition of this conduit is unknown. If the conduit is broken or crushed, it would also have to be replaced. Due to this unknown factor, staff is recommending a contingency of $47,991 to account for the potential replacement of the conduit.",
                        "Vendor Selection",
                        "Bid Data",
                        "Bid Posting Date",
                        "Bid Posted On",
                        "Bid Advertised In",
                        "(City Charter & Muni Code)",
                        "# of Vendors Downloaded",
                        "# of Submittals Received",
                        "Date Publicly Opened",
                        "08/27/2018",
                        "City's Online Bidding Site",
                        "Santa Monica Daily Press",
                        "36",
                        "4",
                        "09/13/2018",
                        "Selection Criteria",
                        "Municipal Code",
                        "SMMC 2.24.072",
                        "Evaluation Criteria",
                        "Price, previous experience, ability to deliver, quality of product and compliance with City specifications",
                        "Best Bidder",
                        "Bales Engineering",
                        "Financial Impacts and Budget Actions",
                        "Staff seeks authority to award a contract with Bales Engineering to provide construction services for the SMO \u2013 Cable Replacement of Airfield Lighting Project.",
                        "Contract Request",
                        "Request Amount",
                        "FY 2018-19 Budget CIP Account #",
                        "Total Contract Amount",
                        "$200,000",
                        "C5704710.689000",
                        "$200,000"
                    ],
                    "department": "Public Works, Civil Engineering",
                    "id": 1011,
                    "meeting_time": 1539131400,
                    "recommendations": [
                        {
                            "agenda_item": 1011,
                            "id": 1011,
                            "recommendation": [
                                "1.     Award Bid #2587 to Bales Engineering Contractors, Inc., a California-based company, to provide construction services for the Santa Monica Airport \u2013 Cable Replacement of Airfield Lighting Project for the Public Works Department;",
                                "2.     Authorize the City Manager to negotiate and execute a contract with Bales Engineering Contractors, Inc., in an amount not to exceed $200,000 (including a $47,991 contingency);",
                                "3.     Authorize the Director of Public Works to issue any necessary change orders to complete additional work within contract authority."
                            ]
                        }
                    ],
                    "sponsors": null,
                    "title": "Award Construction Contract for Santa Monica Airport (SMO) Cable Replacement of Airfield Lighting Project"
                },
```
same issue with agenda. It might be stored as a List.

**Comment by eselkin** [comment]
It is a list, and was changed to be that a long time ago. Not all sections are present and when they are not present, backend was crashing. I wasn't in charge of that. We can look into it again but it's not a requirement.
